dotcom-rendering/src/components/ScrollableCarousel.tsx

```diff
@@ -346,44 +321,22 @@ export const ScrollableCarousel = ({
 			>
 				{children}
 			</ol>
-			{showNavigation && (
-				<div css={buttonStyles}>
-					<Button
-						hideLabel={true}
-						iconSide="left"
-						icon={<SvgChevronLeftSingle />}
-						onClick={() => scrollTo('left')}
-						priority="tertiary"
-						theme={
-							previousButtonEnabled
-								? themeButton
-								: themeButtonDisabled
-						}
-						size="small"
-						disabled={!previousButtonEnabled}
-						// TODO
-						// aria-label="Move stories backwards"
-						// data-link-name="container left chevron"
-					/>
 
-					<Button
-						hideLabel={true}
-						iconSide="left"
-						icon={<SvgChevronRightSingle />}
-						onClick={() => scrollTo('right')}
-						priority="tertiary"
-						theme={
-							nextButtonEnabled
-								? themeButton
-								: themeButtonDisabled
-						}
-						size="small"
-						disabled={!nextButtonEnabled}
-						// TODO
-						// aria-label="Move stories forwards"
-						// data-link-name="container right chevron"
-					/>
-				</div>
+			{showNavigation && (
+				<CarouselNavigationButtons
+					previousButtonEnabled={previousButtonEnabled}
+					nextButtonEnabled={nextButtonEnabled}
+					onClickPreviousButton={() => scrollTo('left')}
+					onClickNextButton={() => scrollTo('right')}
+					dataLinkNamePreviousButton={nestedOphanComponents(
```

Comment:
I can see the values coming through in the DOM - nice! Just wondering, IIUC the point of these values is to be able to get data on what buttons are clicked on the page, but at the moment all carousel buttons have those same values. 

Not sure if it ends up being distinguishable in the datalake, so maybe we should add something which would specify which specific carousel got clicked (if that's possible/desirable)?

Reply:
Ah, should have thought of looking at the click events! Makes sense that it would have a reference to the dataComponent built in, so that all sounds good to me - still worth checking with the data team at some point though 👍